extensions/zerion/package.json

```
@@ -89,5 +148,6 @@
     "fix-lint": "ray lint --fix",
     "lint": "ray lint",
     "publish": "npx @raycast/api@latest publish"
-  }
+  },
+  "version": "2.0.0"
```

```suggestion
  }
```

The `version` field is not needed. Raycast never uses it for any purpose.